spanner/spansql/parser.go

```diff
@@ -1495,6 +1542,98 @@ func (p *parser) parseColumnOptions() (ColumnOptions, *parseError) {
 	return co, nil
 }
 
+func (p *parser) parseDatabaseOptions() (DatabaseOptions, *parseError) {
+	debugf("parseDatabaseOptions: %v", p)
+	/*
+			options_def:
```

Comment:
Same here. I think it should move a tab left.

Reply:
Done